src/Directory.Build.targets

```diff
@@ -12,4 +12,13 @@
   <ItemGroup>
     <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net462'">
+    <!--ImplicitUsings will add this namespace that is not available for NetFX.
+    https://github.com/dotnet/sdk/issues/24146
+    https://github.com/dotnet/runtime/issues/59163
+    https://github.com/dotnet/sdk/issues/22515
+    -->
+    <Using Remove="System.Net.Http" />
+  </ItemGroup>
```

Comment:
I don't have an issue with this but FYI we could also have a build/Common.targets file similar to build/Common.props for this type of shared thing.

Reply:
I was thinking the same! I decided against it because the existing Directory.Build.targets files are mostly empty.
I didn't think this would meet the bar for adding another file to the repo.
If there's a consensus that we should add a Common.targets I would be happy to do that.